src/main/java/com/thoughtworks/gocd/secretmanager/vault/models/SecretConfig.java

```diff
@@ -42,6 +42,8 @@
     public static final List<String> SUPPORTED_AUTH_METHODS = asList(TOKEN_AUTH_METHOD, APPROLE_AUTH_METHOD, CERT_AUTH_METHOD);
     public static final int DEFAULT_CONNECTION_TIMEOUT = 5;
     public static final int DEFAULT_READ_TIMEOUT = 30;
+    public static final int DEFAULT_MAX_RETRIES = 0;
```

Comment:
If you can, I would default this to 2, to match the vault cli

Reply:
https://www.vaultproject.io/docs/commands#vault_max_retries

Reply:
My original thinking was to set it to 0 to match the current behavior (which I believe is the case)